scrapers/scrape_ju.py

```diff
@@ -34,6 +46,8 @@
                 print('-' * 10)
             is_first = False
 
+            sanitize_row(row)
```

Comment:
...and assign it here:

```python
row = sanitize_row(row)
```

Reply:
Should both be addressed in 18f6a3e986a89fb4135b34711fe03b1e6ff0f9fe